dotcom-rendering/src/components/Card/Card.stories.tsx

```diff
@@ -209,34 +209,35 @@ export default {
 	],
 };
 
+/** TODO : Update stories once initial chromatic check has passed on new font size scale */
```

Comment:
Are the stories still to be updated as part of this PR?

Reply:
No, they'll be in a follow up PR. This is to make sure I know that the font changes aren't affecting existing stories